apps/angular/view-transition/src/styles.scss

```diff
@@ -2,3 +2,187 @@
+
+    100% {
+        transform: translateX(0) translateY(0);
+    }
+}
+
+@keyframes slide-to-right {
+    0% {
+        transform: translateX(0) translateY(0) scale(1);
+    }
+
+    100% {
+        transform: translateX(240px) translateY(25px) scale(1.6); // scale makes it harder for pixel precision
+    }
+}
+
+@keyframes slide-to-left-green {
+    0% {
+        transform: translateX(0) translateY(0);
+    }
+
+    100% {
+        transform: translateX(-260px) translateY(720px);
+    }
+}
+
+@keyframes slide-to-right-green {
+    0% {
+        transform: translateX(0) translateY(0) scale(1);
+    }
+
+    100% {
+        transform: translateX(50px) translateY(700px) scale(1.6);
+    }
+}
+
+@keyframes slide-to-left-blue {
+    0% {
+        transform: translateX(0) translateY(0);
+    }
+
+    100% {
+        transform: translateX(-260px) translateY(1220px);
+    }
+}
+
+@keyframes slide-to-right-blue {
+    0% {
+        transform: translateX(0) translateY(0) scale(1);
+    }
+
+    100% {
+        transform: translateX(50px) translateY(1200px) scale(1.6);
+    }
+}
+
+::view-transition-new(profil1a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-2, 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-from-right;
+}
+
+::view-transition-new(angular1a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-2;
+}
+
+::view-transition-new(profil2a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-2, 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-from-right;
+}
+
+::view-transition-new(angular2a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-2;
+}
+
+::view-transition-new(profil3a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-2, 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-from-right;
+}
+
+::view-transition-new(angular3a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-2;
+}
+
+.red::view-transition-old(profil1a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left;
+}
+
+.red::view-transition-old(angular1a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-right;
+}
+
+.red::view-transition-new(angular1) {
+    display: none;
+}
+
+.red::view-transition-new(profil1) {
+    display: none;
+}
+
+.green::view-transition-old(profil2a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-green;
+}
+
+.green::view-transition-old(angular2a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-right-green;
+}
+
+.green::view-transition-new(angular2) {
+    display: none;
+}
+
+.green::view-transition-new(profil2) {
+    display: none;
+}
+
+.blue::view-transition-old(profil3a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-blue;
+}
+
+.blue::view-transition-old(angular3a) {
+    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-right-blue;
+}
+
+.blue::view-transition-new(angular3) {
+    display: none;
+}
+
+.blue::view-transition-new(profil3) {
+    display: none;
+}
```

Comment:
That's what we want to avoid. Because on this example, we only have 3 thumbails, but in a real project we can have a lot more 😅

and also you can write them like:
```css
::view-transition-new(angular1a) {
    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-2;
}

::view-transition-new(profil2a),
::view-transition-new(angular2a) {
    animation: 1500ms cubic-bezier(0.4, 0, 0.2, 1) slide-to-left-2;
}
```

since the css property is the same